14/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="UA">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>14</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="" />
+  </head>
+  <body>
+    <div>
+      <label for="task1"></label>
+      <input type="text" id="task1" placeholder="task1" />
+      <button id="task1_button"></button>
```

Comment:
Nice to have any text in the button it looks bad 
<img width="181" alt="image" src="https://user-images.githubusercontent.com/84010501/200787912-0cf0799d-0925-4b01-ad81-111171d84c4d.png">